<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FarmSetting</class>
 <widget class="QDialog" name="FarmSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>157</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Farm Setting</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>6</y>
     <width>133</width>
     <height>113</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_cpu">
      <property name="text">
       <string>CPU</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_gpu">
      <property name="text">
       <string>Single GPU</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_mgpu">
      <property name="text">
       <string>Multiple GPUs'</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_mgpu">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
